### ISSUE TITLE
Installation fails on Ubuntu 16.06

### ISSUE THREAD

**Issue by NeoTheThird** [anbox/anbox#2]
Make sure you are running the latest version of Anbox before reporting an issue.

**Anbox release (`anbox version`) and system info (`anbox system-info`):** 0.1.0

**Description of the problem:** The installation script exits with a syntax error.

**Expected:** Succsessful installation of Anbox.

**Additional info:** [Log](http://paste.ubuntu.com/24362283/)

**Comment by NeoTheThird** [comment]
It's 16.04 of course.

**Comment by clessley** [comment]
Me too, Ubuntu MATE 64-bit 16.04, using the wget method to get the installer > install > I AGREE > runs but fails after installing upstart session job .. +cat + initctl start anbox is the bad line:

 `/snap/anbox-installer/7/installer.sh: line 172: initctl: command not found`

**Comment by morphis** [comment]
This is interesting. Isn't Ubuntu MATE running a upstart powered user session?

**Comment by lool** [comment]
Hmm the installer should test whether *in* an upstart or systemd session, this would also detect installs over SSH; guessing Ubuntu MATE is running systemd as session manager.

**Comment by morphis** [comment]
@lool yeah that sounds like a good idea. Would be better than the lazy check for > 16.04 I am currently doing to detect systemd --user

**Comment by flexiondotorg** [comment]
@lool @morphis Ubuntu MATE doesn't use upstart or systemd for the session manager.

**Comment by shellkr** [comment]
There are no distros still using Upstart. Ubuntu dropped it in order to adopt systemd on 15.04. I see no reason still having that support except for backward compatibility.

**Comment by NeoTheThird** [comment]
@shellkr I guess that is because Anbox was prototyped on Ubuntu Touch which is using Upstart.

**Comment by morphis** [comment]
@shellkr Ubuntu 16.04 still uses upstart for the user session. It doesn't use `systemd --user` yet. That was introduced with 16.10.

**Comment by sudopluto** [comment]
kde neon also uses systemd user sessions

**Comment by Tirpitz93** [comment]
So, is there a workaround for systems using systemd?

**Comment by sskras** [comment]
Is the issue still present and relevant?

**Comment by Fuseteam** [comment]
well ubuntu 16,04 is now end of life sooooo
